CMSgov/cms-carts-seds
# Update Userinfo component

### Description

Migrate redux `connect` pattern to `useSelector` hook

### Related ticket(s)


CMDCT-3812

---
### How to test

Run `main` locally
- Log in as a state user
- Navigate to `/userinfo`
- Check the page at different widths
- Check the console for errors
- Change to this branch `cmdct-3812` and verify there are no changes to the page

Alternatively, compare the [deployed branch](https://d2k3tvv8jlodnc.cloudfront.net/) with [CARTS dev](https://mdctcartsdev.cms.gov/)

### Notes


I tried to rename this file to `UserInfo` but the AppRoutes tests started failing on the import 🤔 

---
### Pre-review checklist
